GVFS/GVFS.Common/Cleanup/GitCleanupQueue.cs

```diff
@@ -0,0 +1,103 @@
+﻿using GVFS.Common.Tracing;
+using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Threading;
+
+namespace GVFS.Common.Cleanup

```

Comment:
I'm not sold on the word `Cleanup`.  Is Prefetch cleaning anything up or generation of the midx or commit graph?  What about "Optimize" or "Optimization"?

Reply:
I'll use `Maintenance` in the next version.